Touch Bar Simulator/util.swift

```diff
@@ -72,7 +73,7 @@ extension NSWindow {
 		}
 		switch yPositioning {
 		case .top:
-			y = visibleFrame.maxY - frame.height
+			y = min(visibleFrame.maxY - frame.height, screen.frame.maxY - 22 - frame.height)
```

Comment:
Can you add a short comment that explains this logic (Like in the commit message).

And can you move `22` into a variable with a descriptive name?

Reply:
You can use `NSStatusBar.system.thickness` to retrieve the status bar height (`22`).